indico/htdocs/sass/partials/_jquery-indico-locationpicker.scss

```diff
@@ -0,0 +1,186 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+.i-location-field {
+    max-width: 400px;
+
+    .wrapper {
+
+        .typeahead-container,
+        .typeahead-field {
+            display: inline-block;
+            position: relative;
+            z-index: 1030 !important;
```

Comment:
Cause I needed it below 1050 and below 1060 which I use in other places. And I think that the typeahead backdrop was using 1040, so I needed to hide the fields behind it. But since we don't use the backdrop anymore, I guess I could review these values.